client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html

```diff
@@ -693,13 +684,11 @@
                             <oui-action-menu-item
                                 data-ng-if="!ola.isActivated()"
                                 data-text="{{:: 'server_ola_activate' | translate }}"
-                                aria-label="{{:: 'server_ola_activate' | translate }}"
                                 data-ui-sref="app.dedicated.server.interfaces">
                             </oui-action-menu-item>
                             <oui-action-menu-item
                                 data-ng-if="ola.isActivated()"
                                 data-text="{{:: 'server_ola_configure' | translate }}"
```

Comment:
`data-text` is deprecated on `oui-action-menu-item`

Reply:
I've remove all of them to a simple `<span data-translate=""></span>` :)